docs/getting-started.rst

```diff
@@ -118,15 +96,13 @@ Next steps
 Once the JDBC driver is set up, you probably want to :ref:`connect to CrateDB
 <connect>`.
 
-.. _Bintray file navigator: https://bintray.com/crate/crate/crate-jdbc/view/files/io/crate/crate-jdbc-standalone
-.. _Bintray overview page: https://bintray.com/crate/crate/crate-jdbc
-.. _Bintray: https://bintray.com/crate/crate/crate-jdbc
+.. _Maven overview page: https://mvnrepository.com/artifact/io.crate/crate-jdbc
```

Comment:
Sorry I might have confused you with mvnrepository.com - it's a third party tool that's not affiliated with maven central. Better to use the official search page at `https://search.maven.org/`, or the direct link to `https://repo1.maven.org/maven2/io/...` (note crate is not on that list until we've done a release).